src/docfx/cli/CommandLineOptions.cs

```diff
@@ -11,6 +11,7 @@ internal class CommandLineOptions
         public bool Legacy;
         public string GitToken;
         public string GitHubToken;
+        public string Locale;
```

Comment:
Can this support multiple locales in one build? Feature request from community likes to switch between languages for an article. #272 #2025

Reply:
We should support at least these cases:

- build just English: `docfx build`
- build English and all other locales: `docfx build --all-locales`
- build designated locales: `docfx build --locale zh-hant --locale zh-hans --locale en-us`

For this to work, we need:

- `--locale` allows multiple
- `--all-locale`
- a `locales` config entry